src/expressions.md

```diff
@@ -327,96 +429,120 @@ let y = 0...10;
 assert_eq!(x, y);
 ```
 
-## Unary operator expressions
+## Operator expressions
+
+Operators are defined for built in types by the Rust language. Operators will
+panic when they overflow when compiled in debug mode. Many of the following
+operators can be overloaded using traits in `std::ops` or `std::cmp`. Note that
+the traits in `std::cmp` are used more generally to show how a type may be
```

Comment:
Here, and other places, should I link to `std` documentation?
Should I mention that these are also in `core`?